src/SSOData/SharedClaimsTrait.php

```diff
@@ -0,0 +1,32 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Trait to with shared claims of a JWT token.
+ *
+ * PHP version 7.4
+ *
+ * @category  Token
+ * @copyright 2017-2022 Staffbase, GmbH.
+ * @author    Daniel Grosse
+ * @license   http://www.apache.org/licenses/LICENSE-2.0
+ * @link      https://github.com/staffbase/plugins-sdk-php
+ */
+
+namespace Staffbase\plugins\sdk\SSOData;
+
+/**
+ * Trait to with shared claims of a JWT token.
+ */
+trait SharedClaimsTrait
+{
+    public static string $CLAIM_AUDIENCE                    = 'aud';
+    public static string $CLAIM_EXPIRE_AT                   = 'exp';
+    public static string $CLAIM_JWT_ID                      = 'jti';
+    public static string $CLAIM_ISSUED_AT                   = 'iat';
+    public static string $CLAIM_ISSUER                      = 'iss';
+    public static string $CLAIM_NOT_BEFORE                  = 'nbf';
+    public static string $CLAIM_SUBJECT                     = 'sub';
+
+    public static string $CLAIM_USER_ROLE                   = 'role';
```

Comment:
These where formerly constants but traits don't allow them, so I converted them to this static properties. So I would leave the  casing.

Reply:
And why don't you use an interface instead of a trait?

Example:
```php
interface constantsProvider {
    const CLAIM_BRANCH_ID              = 'branch_id';

}

class A implements constantsProvider {

    public function __construct()
    {
        echo self::CLAIM_BRANCH_ID;
    }
}

class B implements constantsProvider {

    public function __construct()
    {
        echo self::CLAIM_BRANCH_ID;
    }
}

new A();
//output: branch_id

new B();
//output: branch_id
```